packages/webpacker/package.json

```diff
@@ -11,54 +11,45 @@
   "license": "AGPL-3.0",
   "main": "index.js",
   "dependencies": {
-    "@babel/core": "^7.15.5",
-    "@babel/eslint-parser": "^7.16.5",
-    "@babel/plugin-transform-classes": "^7.16.7",
-    "@babel/plugin-transform-regenerator": "^7.13.15",
-    "@babel/plugin-transform-runtime": "^7.15.0",
-    "@babel/preset-env": "^7.15.6",
-    "@babel/preset-react": "^7.12.13",
-    "@babel/runtime": "^7.15.4",
-    "@rails/ujs": "^6.1.3",
+    "@babel/core": "^7.21.3",
+    "@babel/eslint-parser": "^7.21.3",
+    "@babel/plugin-transform-classes": "^7.21.0",
+    "@babel/plugin-transform-regenerator": "^7.20.5",
+    "@babel/plugin-transform-runtime": "^7.21.0",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/runtime": "^7.21.0",
+    "@rails/ujs": "^6.1.7",
     "@tailwindcss/typography": "^0.5.2",
-    "autoprefixer": "^10.4.1",
-    "babel-loader": "^8.2.2",
-    "compression-webpack-plugin": "^9.0.0",
-    "css-loader": "^6.5.1",
-    "expose-loader": "^2.0.0",
-    "glob": "^7.2.0",
+    "autoprefixer": "^10.4.14",
+    "babel-loader": "^9.0.0",
+    "compression-webpack-plugin": "^9.0.0 || ^10.0.0",
```

Comment:
Here we still have the double version dependency.

Is there any reason maintaining the v9 support or could we only require v10?